### ISSUE TITLE
Caldav support

### ISSUE THREAD

**Issue by SamirSaidani** [nanawel/our-shopping-list#2]
Hi,

Thanks for this nice software ! 
As a feature request, I suggest to implement a caldav support, to be able to connect "our-shopping-list" to the nextcloud caldav server, or any caldav-compatible server. Caldav is a widely used opensource protocol (see for instance Tasks - Android, Nextcloud, SoGo, etc.).

**Comment by nanawel** [comment]
Hi @SamirSaidani,

It's been a while but this feature has not been forgotten yet, don't worry.
I was thinking initially that you wanted OSL to act as a CalDAV _server_, but I realize that you wanted it instead to be the _client_, in order to be able to store lists and items on a CalDAV-compatible server like Nextcloud.

Unfortunately, if the first option is feasible (even if quite complex, primarily because I cannot find any stable and maintained server library in JS), the second one basically means that the whole application has to be rewritten and largely complexified, which is not my objective at the moment.

Am I misunderstanding your proposal here?

**Comment by SamirSaidani** [comment]
You're correct, I was proposing OSL to act as a caldav client, like tasks.org, thunderbird, etc.
